### PR TITLE
[zh] Resync "Configure Multiple Schedulers" #31502

### PR DESCRIPTION
**Pull request by PriyanshuAhlawat**
Updated the 'Enable leader election' section in Chinese version .

### CODE REVIEW

**Comment by jihoon-seo** [comment]
According to the issue #31502,
lines that are related to `--scheduler-name` should be also updated.

And according to [this comment](https://github.com/kubernetes/website/pull/30486#issuecomment-969771556),
> Thank you for proposing this fix. However, the Chinese l10n team will not merge partial fixes as per stated in the guideline. To find out what are the upstream changes to be synced, do this on the main branch:
> 
> ```
> ./scripts/lsync.sh content/zh/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md
> ```

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 1]
```diff
@@ -192,11 +192,11 @@ First, update the following fields in your YAML file:
...
 
-首先，更新上述 Deployment YAML（my-scheduler.yaml）文件中的以下字段：
+更新 KubeSchedulerConfiguration 中的以下字段 `my-scheduler-config` YAML 文件中的 ConfigMap
 
-* `--leader-elect=true`
-* `--lock-object-namespace=<lock-object-namespace>`
-* `--lock-object-name=<lock-object-name>`
+* `leaderElection.leaderElect`至`true`
+* `leaderElection.resourceNamespace`至`<lock-object-namespace>`
+* `leaderElection.resourceName`至`<lock-object-name>`
```
Comment (sftim) [inline content/zh/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:200]: Where's the English version of this text? I don't see it. Usually there would be an HTML comment holding the English text.

Comment (PriyanshuAhlawat) [reply content/zh/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:200]: sorry for missing that , I am a beginner and just started contributing to k8s.

**Comment by tengqm** [comment]
/close
No response from the author for over a month. There is merge conflict to be resolved.
Please feel free to reopen and resume the work.

**Pull request closed without merging**